src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -1291,7 +1291,7 @@ public String getValue() {
 
             @Override
             public String getDisplayName() {
-                return null;
+                return "";
```

Comment:
With the new parent pom, findbugs is integrated. With the current version of the core (1.566), the plugin builds ok and findbugs does not find issues.

However, if this is run against the latest core version, findbugs starts complaining because the method getDisplayName() in the superclass is annotated with @Nonnull. It is important to avoid this, so that the plugin compatibility test works correctly.